pyquil/paulis.py

```diff
@@ -833,7 +836,7 @@ def exponentiate(term: PauliTerm):
     return exponential_map(term)(1.0)
 
 
-def exponential_map(term):
+def exponential_map(term: PauliTerm) -> Callable[[float], Program]:
```

Comment:
```suggestion
def exponential_map(term: PauliTerm) -> Callable[float]:
```

Reply:
pretty sure the resultant function just takes a `float`

Reply:
It returns a `Program` still.

Reply:
my mistake, misunderstood the `Callable` syntax